### DISCUSSION TITLE
Feature request: Sensors SWAP DISK

### DISCUSSION THREAD

**Discussion by milandzuris** [dougiteixeira/proxmoxve#122] [Feature request] [answered]
Hi i need Sensor for Swap Used in GB and Disk Free in GB THX

**Comment by dougiteixeira** [comment, marked as answer]
It was added in version 2.0.0.b0 (beta).

**Comment by milandzuris** [comment]
where is it DISK free in GB? and swap free in gb?? @dougiteixeira 
![image](https://github.com/dougiteixeira/proxmoxve/assets/42905545/2fde7690-f67c-4ae8-97c4-90977ef3d0a8)
